ebookmaker/EbookMaker.py

```diff
@@ -30,7 +30,7 @@
 from libgutenberg.Logger import debug, warning, error, exception
 from libgutenberg import Logger, DublinCore
 from libgutenberg import MediaTypes as mt
-from libgutenberg import Cover
+#from libgutenberg import Cover
```

Comment:
can't do hasattr(Cover, 'cairo') without importing Cover!

Reply:
But if we import Cover, Cover would import cairo which causes the errors. We cannot modify files in libgutenberg for this problem right?